tests/integration/suite/daprd/workflow/loadbalance/config.go

```diff
@@ -0,0 +1,43 @@
+/*
+Copyright 2025 The Dapr Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://wwb.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package loadbalance
+
+import (
+	"testing"
+
+	"github.com/dapr/dapr/tests/integration/framework/process/daprd"
+	"github.com/dapr/dapr/tests/integration/framework/process/workflow"
+)
+
+func newClusteredDeployment(t *testing.T, daprds int) *workflow.Workflow {
```

Comment:
Can you write a test that involves a case where there is a dapr or 2 not using the clustered deployment and one or 2 using the clustered deployment and ensure we dont have any collisions with state, wf names, etc? Just want to ensure there are no issues with a mixed deployment model. I would like to ensure we dont have any collisions/issues. This might need to be something documented explicitly.

Reply:
added a conflict test